definitions/features/pulpcore.rb

```diff
@@ -12,7 +12,7 @@ class Features::Pulpcore < ForemanMaintain::Feature
 
   def services
     self.class.pulpcore_common_services + configured_workers + [
-      system_service('rh-redis5-redis', 5),
+      system_service(feature(:redis).service_name, 5),
```

Comment:
should this instead be
```
      feature(:redis).services
```

from https://github.com/theforeman/foreman_maintain/blob/238cbf6de91814df3b316724d221e184d73282de/definitions/features/redis.rb#L11-L13

but then the priority is different?!

Reply:
The redis is required for pulpcore and hence I think it should be priority 5 in redis feature as well

Reply:
Okay, will update then. Thanks